lemminx-maven/src/test/resources/pom-with-properties.xml

```diff
@@ -11,4 +11,9 @@
 	<properties>
 		<myProperty>$</myProperty>
 	</properties>
+	<dependencies>
+		<dependency>
+		<groupId>${myProperty}</groupId>
```

Comment:
Try some test with 2 properties, like `<groupId>group.${myProperty}.blah.${project.build.directory}</groupId>`

Reply:
For this case, I'm not entirely sure it's possible to differentiate when the mouse is hovering over the first property or the second property. I believe the `Range` of the hover request for onText contains the entire range of text between the enclosing tags. (I should know better.. I'm the one who implemented onText  :laughing:)

If this is the case, I'll just aggregate the properties into a single hover.
WDYT?

Reply:
request.getOffset() gives you the precise mouse loation, so you can differentiate.

Reply:
Ok thanks, will use that.